.github/workflows/agw-build-publish-container.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: AGW Build & Publish Container
+
+on:
+  push:
+    branches:
+      - master
+      - 'v1.*'
+  pull_request_target:
+    types: [ opened, reopened, synchronize ]
+    branches:
+      - master
+      - 'v1.*'
+
+jobs:
+
+  collect-is-successful:
+    runs-on: ubuntu-latest
+    if: success()
+    needs: build-containers
```

Comment:
Why not check on the status of the GHz build?

Reply:
Because GHZ are not needed for the magma product and just some test and debugging images. So they are largely irrelevant. (Besides they also have loads of duplication and are a possible candidate for further cleanup.)